config/application.rb

```diff
@@ -11,5 +11,6 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.middleware.use Rack::Deflater
```

Comment:
With the current production configuration it isn't. Is there any advantage of setting up compression in the frontend server instead of the application?